docs/articles/basics/first_bot.md

```diff
@@ -111,12 +111,15 @@ We'll now create a new `DiscordClient` instance in our brand new asynchronous me
 
 Create a new variable in `MainAsync` and assign it a new `DiscordClient` instance, then pass an instance of `DiscordConfiguration` to its constructor.
 Create an object initializer for `DiscordConfiguration` and populate the `Token` property with your bot token then set the `TokenType` property to `TokenType.Bot`.
+Next add the `Intents` Property and Populated it with the @DSharpPlus.DiscordIntents.GuildMessages and @DSharpPlus.DiscordIntents.DirectMessages values.  These Intents 
+are required for certain Events to be fired.  Please visit this [article](xref:beyond_basics_intents) for more information.
 
 ```cs
 var discord = new DiscordClient(new DiscordConfiguration()
 {
     Token = "My First Token",
-    TokenType = TokenType.Bot       
+    TokenType = TokenType.Bot,
+    Intents = DiscordIntents.GuildMessages | DiscordIntents.DirectMessages       
```

Comment:
The config has it defaulted to `AllUnprivileged` by default, so this isn't necessary. Just having it be `DiscordIntents.AllUnprivileged` should they can get an idea of them should be fine.